src/SFA.DAS.Commitments.Support.SubSite/Enums/CommitmentRequestStatus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.ComponentModel;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Enums
+{
+    public enum RequestStatus
```

Comment:
The source RequestStatus in SFA.DAS.EmployerCommitments.Application has been updated recently with extra constants related to transfers, so this enum needs updating.